extension/content.js

```diff
@@ -344,6 +344,21 @@ function addFilterCommentsByYou() {
 	}
 }
 
+function addProjectNewLink() {
+	const projectNewLinkId = 'refined-github-project-new-link';
+	const projectNewLink = `<a href="/${repoUrl}/projects/new" class="btn btn-sm" id="${projectNewLinkId}">Add a project</a>`;
+	if ($(`#${projectNewLinkId}`).length === 0) {
+		$('#projects-feature').siblings('p.note').after(projectNewLink);
```

Comment:
@lukaszklis @paulmolluzzo Please see my other comment: https://github.com/sindresorhus/refined-github/pull/411#discussion_r115561892 I've posted it just after your commit.

Pasting it here: 

The ideal method for selecting would [`$.fn.nextAll()`](https://api.jquery.com/nextAll/):

```js
$('#projects-feature').nextAll('p.note')
```

I've tested it on the Settings page and it finds the needed element. We could also use a CSS selector:
```js
$('#projects-feature ~ p.note')
```